### PR TITLE
Real fix locals not a scam

### PR DESCRIPTION
**Pull request by Hexcede**

### CODE REVIEW

File: Locals.lua [hunk 1]
```diff
@@ -0,0 +1,23 @@
+local getLocals = "getLocals" -- fix bug
```
Comment (4812571) [inline Locals.lua:1]: How does this fix the bug?

File: Locals.lua [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+function Locals.getLocals(this)
```
Comment (4812571) [inline Locals.lua:3]: This is an alias for rawset, effectively

File: Locals.lua [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    return this
+end
+function Locals.local(this, name)
```
Comment (4812571) [inline Locals.lua:6]: this is just get

File: Locals.lua [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    return this[name]
+end
+function Locals.set(this, name, value)
```
Comment (4812571) [inline Locals.lua:9]: this is just set

File: Locals.lua [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    this[name] = value
+    return value, name
+end
+function Locals.new()
+    return setmetatable({
```
Comment (4812571) [inline Locals.lua:14]: metatable backwards?!

File: Locals.lua [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+        getLocals = Locals[getLocals];
+        ["local"] = Locals["local"],
+        __index = getLocals
+    }, Locals)
+end
+
+local locals = Locals.new()
+locals.loclas = Locals
```
Comment (4812571) [inline Locals.lua:22]: A monad is a monoid in the category of endofunctors. What's the problem?!

File: README.md [hunk 7]
```diff
@@ -1,12 +0,0 @@
-# Voltz OSS
-
-Voltz, open-sourced.
-
----
-
-## Installing tools
```
Comment (4812571) [inline README.md:7]: This was neccesary

File: README.md [hunk 8]
```diff
@@ -1,12 +0,0 @@
...
-
-1. Install Rust via [rustup](https://rustup.rs/)
-2. Install foreman via `cargo install foreman`
-3. Run `foreman install` within the project root to download and install necessary tools
```
Comment (4812571) [inline README.md:11]: This was necessary

File: default.project.json [hunk 9]
```diff
@@ -1,31 +0,0 @@
-{
```
Comment (4812571) [inline default.project.json:1]: This was necessary

File: foreman.toml [hunk 10]
```diff
@@ -1,3 +0,0 @@
-[tools]
```
Comment (4812571) [inline foreman.toml:1]: This was necessary

**Review by 4812571 — CHANGES_REQUESTED** [review]
This is the dumbest commit I have ever read.

File: Locals.lua (hunk 1 above)
Comment (Hexcede) [reply Locals.lua:1]: it does

File: Locals.lua (hunk 2 above)
Comment (Hexcede) [reply Locals.lua:3]: yes

File: Locals.lua (hunk 3 above)
Comment (Hexcede) [reply Locals.lua:6]: no its rawset

File: Locals.lua (hunk 4 above)
Comment (Hexcede) [reply Locals.lua:9]: no its rawget

File: Locals.lua (hunk 5 above)
Comment (Hexcede) [reply Locals.lua:14]: no

File: Locals.lua (hunk 6 above)
Comment (Hexcede) [reply Locals.lua:22]: ---

File: README.md (hunk 8 above)
Comment (Hexcede) [reply README.md:11]: yes

File: default.project.json (hunk 9 above)
Comment (Hexcede) [reply default.project.json:1]: yes

File: foreman.toml (hunk 10 above)
Comment (Hexcede) [reply foreman.toml:1]: yes

**Comment by Hexcede** [comment]
closing due to:
---

**Pull request closed without merging**